### ISSUE TITLE
Google Drive Extension Not Working

### ISSUE THREAD

**Issue by PhantomNimbi** [aniyomiorg/aniyomi-extensions#1901]
### Source information and language

Google Drive 13.6

### Steps to reproduce

1. Download and install Google drive extension
2. Setup the Google Drive folder(s) for it to use 
3. Attempt to view the videos

### Expected behavior

It should allow the user to access the content of the Google drive folder.

### Actual behavior

It tells the user to.login via web view and gets blocked by Google's OAuth policies when attempting to login.

### Aniyomi version

0.12.3.10

### Android version

Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3.10](https://github.com/aniyomiorg/aniyomi/releases/latest)**
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://aniyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/aniyomiorg/aniyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.
